cognite/neat/_session/_template.py

```diff
@@ -174,6 +175,18 @@ def data_product_model(
             self._state.last_reference = last_rules
         return issues
 
+    def conceptual_model(self, io: Any) -> None:
```

Comment:
We discussed this one in Japan, I remember one of the confusions we were afraid of was the 33 concepts in core could be a lot confusion.

```suggestion
    def conceptual_model(self, io: Any, base_model: Literal["CogniteCore"] = "CogniteCore", n_concepts=5) -> None:
```
One idea was to have a signature like above, and use a parameter `n_concepts=5` to select the number of concepts to include. For example, 5 would only include CogniteAsset, CogniteEquipment, CogniteFile, CogniteTimeSeries, CogniteActivity.